scene/main/node.cpp

```diff
@@ -2147,30 +2147,42 @@ int Node::get_persistent_group_count() const {
 	return count;
 }
 
-void Node::_print_tree_pretty(const String &prefix, const bool last) {
+void Node::print_tree_pretty() {
+	print_line(_get_tree_string_pretty("", true));
+}
+
+void Node::print_tree() {
+	print_line(_get_tree_string(this));
+}
+
+String Node::_get_tree_string_pretty(const String &prefix, const bool last) {
 	String new_prefix = last ? String::utf8(" ┖╴") : String::utf8(" ┠╴");
-	print_line(prefix + new_prefix + String(get_name()));
 	_update_children_cache();
-	for (uint32_t i = 0; i < data.children_cache.size(); i++) {
+	String return_tree = "";
+	return_tree += prefix + new_prefix + String(get_name()) + "\n";
+	for (int i = 0; i < data.children.size(); i++) {
```

Comment:
You're still using children here, see the code you're replacing

Reply:
As I said above, I'd recommend restoring all the original code and then adding your changes again to avoid overwriting or ignoring important code